File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int n=nums.length;
        int[] r=new int[n];
        for(int i=0;i<n;i++){
            int p=1;
            for(int j=0;j<n;j++){
                if(i!=j){
                    p*=nums[j];
                }
            }
            r[i]=p;
        }
        return r;
    }
}  

File: Data Structures & Algorithms/maximum-subarray/submission-0.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        n=len(nums)
        res=nums[0]
        for i in range(n):
            c=0
            for j in range(i,n):
                c+=nums[j]
                res=max(res,c)
        return res
